### ISSUE TITLE
Grunt 0.4 Release

### ISSUE THREAD

**Issue by tkellen** [punkave/grunt-dox#8]
I'm posting this issue to let you know that we will be publishing Grunt 0.4 on Monday, February 18th.

If your plugin is not already Grunt 0.4 compatible, would you please consider updating it? For an overview of what's changed, please see our [migration guide](https://github.com/gruntjs/grunt/wiki/Upgrading-from-0.3-to-0.4).

If you'd like to develop against the final version of Grunt before Monday, please specify `"grunt": "0.4.0rc8"` as a devDependency in your project. After Monday's release, you'll be able to use `"grunt": "~0.4.0"` to actually publish your plugin. If you depend on any plugins from the grunt-contrib series, please see our [list of release candidates](https://github.com/gruntjs/grunt/wiki/Plugin-Release-Candidate-Versions) for compatible versions. All of these will be updated to final status when Grunt 0.4 is published.

Also, in an effort to reduce duplication of effort and fragmentation in the developer community, could you review the [grunt-contrib](http://github.com/gruntjs) series of plugins to see if any of your functionality overlaps significantly with them?  Grunt-contrib is community maintained with 40+ contributors&mdash;we'd love to discuss any additions you'd like to make.

Finally, we're working on a new task format that doesn't depend on Grunt: it's called [node-task](/tkellen/node-task).  Once this is complete, there will be one more conversion, and then we'll never ask you to upgrade your plugins to support our changes again.  Until that happens, thanks for bearing with us!

If you have any questions about how to proceed, please respond here, or join us in [#grunt on irc.freenode.net](irc://irc.freenode.net/#grunt).

Thanks, we really appreciate your work!

**Comment by jeremyckahn** [comment]
I take this issue to mean that grunt-dox is not compatible with Grunt 0.4.  Are there plans to update it?  @mattmcmanus, would you like some help?

**Comment by mattmcmanus** [comment]
To be honest, I just haven't had to chance to look into this yet. There is a pull request #6 that is a little old but has some support that I just haven't had a chance to verify yet. I'll make it a priority to get to this in the next day or so.

**Comment by mattmcmanus** [comment]
I just merged in #6 and have confirmed the task works un Grunt 0.4. Thanks for the nudge @jeremyckahn and @tkellen

**Comment by jeremyckahn** [comment]
Thanks for merging!  I just popped this into [Rekapi](https://github.com/jeremyckahn/rekapi/) to try it out, as I am currently updating the project's docs to be generated by `dox-foundation`.  However, I am getting an error.  Here's the relevant snippet from my `Gruntfile.js`:

``` javascript
dox: {                                                                                                                                                                                                                                    
  options: {                                                                                                                                                                                                                              
    title: 'Rekapi'                                                                                                                                                                                                                       
  },                                                                                                                                                                                                                                      
  files: {                                                                                                                                                                                                                                
    src: ['src/rekapi.core.js'],                                                                                                                                                                                                          
    dest: 'dist/doc/core'                                                                                                                                                                                                                 
  }                                                                                                                                                                                                                                       
}   
```

And I see this error when I run `grunt dox`:

```
$: grunt dox
Running "dox:files" (dox) task
>> ERROR:  Error: Command failed: 
>>   error: unknown option `--source'
```

I logged out what's being `exec`ed [here](https://github.com/punkave/grunt-dox/blob/master/tasks/dox.js#L43):

```
/Users/jeremyckahn/Sites/lib/rekapi/node_modules/grunt-dox/node_modules/.bin/dox-foundation --source src/rekapi.core.js --target dist/doc/core --title Rekapi
```

I tried to figure out, but I'm not sure what's going on.  When I use `dox-foundation`, I use it like this:

```
$ dox-foundation < myfile.js > myfile.html
```

...In other words, I never used the `--source` switch, so I have nothing to compare it to.  Is this method still supported?  I see [it was taken out of the README](https://github.com/punkave/dox-foundation/commit/448d679e3790e3dce14db9261e638baea8c2f546).  This might be a PEBKAC issue, but do you have any ideas on what the issue might be?

I am running OS X 10.8.3.

**Comment by mattmcmanus** [comment]
Alrighty. I recently merged in a big pull request that had the dox-foundation version in the package.json back at 0.2. I just pushed 0.4.1 of grunt-dox which fixed that. Can you try again?

**Comment by jeremyckahn** [comment]
I just tested out 0.4.1, it works great!  Thanks so much!  I also appreciate the updated CSS, it looks awesome. :)
